CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.5.1] - 2021-08-04
```

Comment:
Date is going to need updating.

Reply:
```suggestion
## [2.5.1] - 2021-08-10
```

Reply:
There should also be a link URL added to the bottom of the file that goes with the `[2.5.1]`.